src/mds/Server.cc

```diff
@@ -1447,6 +1447,18 @@ void Server::handle_client_reconnect(const cref_t<MClientReconnect> &m)
     return;
   }
 
+  if(mds->mdsmap->test_flag(CEPH_MDSMAP_REFUSE_CLIENT_SESSION)) {
+    mds->clog->warn() << "client could not reconnect as" 
+                         " file system flag refuse_client_session is set";
+    dout(7) << "client cannot reconnect when file system flag"
```

Comment:
This can be dout(0) too? Similar to the log message in Server::handle_client_session

Reply:
Ah, right. I was experimenting something and forgot to change it. Will do it, thanks!

Reply:
Done.